src/main/webapp/app/components/Header.tsx

```diff
@@ -42,6 +42,27 @@ type SubpageLink = {
   link: string;
 };
 
+const checkIfActive = (title: string, subPages: SubpageLink[]) => {
```

Comment:
```suggestion
const checkIfNavLinkIsActive = (title: string, subPages: SubpageLink[]) => {
```
Let's have a more clear naming